src/components/SideNav.js

```diff
@@ -83,21 +97,21 @@ export default function SideNav({ setKeyword, children }) {
       name: "Add Rating",
       href: "/rate",
       initial: "+",
-      current: false,
+      current: pathName === "/rate",
```

Comment:
Looking good, are these still needed, because the check is done on line `60` with `href`?

Reply:
I can see that we have 2 navigation menus. One is for the public routes and the other is for secure routes.
Have made changes to both
The secure routes were declared inside the function. So have not moved it out of the function.

Reply:
Oh yes you are correct, these should be both done the same way - that was my mistake

I think maybe both can be moved inside the function and have then then the `map` can be removed?

Sorry for the confusion and extra changes

Reply:
haha. no worries. I think then my first commit is the right way to go about this issue

Reply:
Done

Reply:
thank you 👍